From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@electronjs.org>
Date: Thu, 16 Jun 2022 16:22:58 -0400
Subject: don't freeze v8 flags after initialization

The --freeze-flags-after-init V8 flag breaks Electron's node integration, so
for now don't set that flag.

diff --git a/content/renderer/render_process_impl.cc b/content/renderer/render_process_impl.cc
index 02b4f803369202a81f2d54f3b35aef04be13dfa2..243d5c790be1b7f83f7ca56942dc0699b1a3e9d0 100644
--- a/content/renderer/render_process_impl.cc
+++ b/content/renderer/render_process_impl.cc
@@ -222,8 +222,6 @@ RenderProcessImpl::RenderProcessImpl()
   SetV8FlagIfNotFeature(features::kWebAssemblyDynamicTiering,
                         "--no-wasm-dynamic-tiering");
 
-  v8::V8::SetFlagsFromString("--freeze-flags-after-init");
-
 #if (BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_CHROMEOS)) && defined(ARCH_CPU_X86_64)
   if (base::FeatureList::IsEnabled(features::kWebAssemblyTrapHandler)) {
     if (command_line->HasSwitch(switches::kEnableCrashpad) ||
